### PR TITLE
added tree for rbx

### PR DESCRIPTION
**Pull request by mdesco**

### CODE REVIEW

File: tree_for_rbx_flow.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+fi
+
+echo "tractoflow folder: ${d}"
+echo "Output folder: ${o}"
+
+echo "Building tree..."
+cd $d
+for i in *[!{FRF}]; 
+do
+   echo $i
```
Comment (GuillaumeTh) [inline tree_for_rbx_flow.sh:24]: Please remove this echo

File: tree_for_rbx_flow.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+usage() { echo "$(basename $0) [-d tractoflow/results] [-o output]" 1>&2; exit 1; }
```
Comment (GuillaumeTh) [inline tree_for_rbx_flow.sh]: Can you use -t instead of -d for tractoflow option

**Review by GuillaumeTh — CHANGES_REQUESTED** [review]
I tested and it works well. I will merge after the minor comments.

File: tree_for_rbx_flow.sh (hunk 1 above)
Comment (mdesco) [reply tree_for_rbx_flow.sh:24]: Can I keep it? I like to see that all subjects were read correctly... It's "reassuring", ok?

Comment (GuillaumeTh) [reply tree_for_rbx_flow.sh:24]: I don like this but I think it is ok. Maybe just add a small echo message like: Data processed or something like this. Just to know that it is the subjects used.

**Review by GuillaumeTh — APPROVED** [review]

**Pull request merged**